doc/SAI_1.6.5_ReleaseNotes.md

```diff
@@ -0,0 +1,539 @@
+In this release, API for TTL and QoS treatment during MPLS encap and decap has been added.
+
+```
+SAI_NEXT_HOP_ATTR_DECREMENT_TTL
+SAI_NEXT_HOP_ATTR_OUTSEG_TYPE
+SAI_NEXT_HOP_ATTR_OUTSEG_TTL_MODE
+SAI_NEXT_HOP_ATTR_OUTSEG_TTL_VALUE
+SAI_NEXT_HOP_ATTR_OUTSEG_EXP_MODE
+SAI_NEXT_HOP_ATTR_OUTSEG_EXP_VALUE
+SAI_NEXT_HOP_ATTR_QOS_TC_AND_COLOR_TO_MPLS_EXP_MAP
+
+```
+
+### sainexthopgroup.h	
+Provides next hop group functions. A router can have many next hops. In case of unicast, a packet can be spread among a group of next hops. Next hop group contains a group of next hops.
+In this release, the below attributes are added in API along with enhancements. 
+
+1)Object index in the fine grain ECMP table. Index specifying the strict member's order.<br>
+2)Allowed value range for is from 0 to SAI_NEXT_HOP_GROUP_ATTR_REAL_SIZE - 1. <br>
+3)Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_FINE_GRAIN_ECMP. <br>
+4)Object's sequence ID for enforcing the members' order. Loose index specifying the member's order. <br>
+5)The index is not strict allowing for the missing IDs in a sequence. <br>
+6)It's driver's job to translate the sequence IDs to the real indexes in the group. Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_DYNAMIC_ORDERED_ECMP. <br>
+
+```
+SAI_NEXT_HOP_GROUP_ATTR_CONFIGURED_SIZE
+SAI_NEXT_HOP_GROUP_ATTR_REAL_SIZE
+   
+```
+### saiport.h	
+SAI port provides following functions:
+
+Provides physical port state manipulation as well as state (UP/DOWN) notifications.
+Obtain physical lane information for a SAI port
+Setup the lane mode for a SAI port
+
+In this release, the below attributes are added in API along with enhancements. 
+
+1)Attribute data for #SAI_PORT_ATTR_INTERFACE_TYPE. Used for selecting electrical interface with specific electrical pin and signal quality. <br>
+2)Port bind point for ingress/egress ACL object. Bind (or unbind) an ingress/egress ACL table or ACL group on a port. Enable/Update ingress/egress ACL table or ACL group filtering by assigning the list of valid object id. Disable ingress/egress filtering by assigning SAI_NULL_OBJECT_ID in the attribute value. <br>
+3)Port bind point for ingress MACsec ACL object. Bind (or unbind) an ingress MACsec ACL table on a port. Enable/Update ingress MACsec ACL table filtering by assigning the list of valid object id. Disable ingress filtering by assigning SAI_NULL_OBJECT_ID in the attribute value. <br>
+4)Link training failure status and error codes port stat PRBS error counts and list of port connector attributes are added. <br>
+
+```
+SAI_PORT_ATTR_INGRESS_MACSEC_ACL
+SAI_PORT_ATTR_EGRESS_MACSEC_ACL
+SAI_PORT_ATTR_MACSEC_PORT_LIST
+

```

Comment:
Will you add the system port changes (and other saiport.h changes) from PR1080?https://github.com/opencomputeproject/SAI/pull/1081

    /**
     * @brief System port for the port
     *
     * @type sai_object_id_t
     * @flags READ_ONLY
     * @objects SAI_OBJECT_TYPE_SYSTEM_PORT
     */
    SAI_PORT_ATTR_SYSTEM_PORT,